packages/devtools_app/lib/src/profiler/cpu_profile_model.dart

```diff
@@ -407,3 +423,47 @@ int stackFrameIdCompare(String a, String b) {
     throw error;
   }
 }
+
+class CpuProfileStore {
+  final _store = <TimeRange, CpuProfileData>{};
+
+  CpuProfileData storedProfile(TimeRange time) {
```

Comment:
nit: maybe call this lookupProfile and note that it will be null if the profile isn't available? Took me a second reading it to realize what storedProfile meant.

Reply:
done